bash_scripting_101/bash_scripting_101.md

```diff
@@ -37,241 +39,443 @@ script: https://kit.fontawesome.com/83b2343bd4.js
 
-![How to locate the terminal in Spotlight on a Mac.](media/terminal_in_mac.png)
 
-- On a Mac machine, bash is the default login shell in the Terminal command line program.
-- Type "terminal" in your Mac spotlight search to pull up this program for the first time (though it might already launch in the dock on your machine).
-- It should be noted that Macs with the Catalina Operating System or later are running zsh instead of bash. That said, zsh is [functionally a later and greater version of bash with some key differences noted](https://medium.com/@harrison.miller13_28580/bash-vs-z-shell-a-tale-of-two-command-line-shells-c65bb66e4658).
 
-### Windows Installation
+## Navigating in bash
 
-![Confirm these Windows settings to enable shell scripting capabilities.](media/on_windows.png)
+If you are new to bash scripting, it can take some adjustment to navigate by typing text. Once you get the hang of a few basic commands, this can be an easier way to deal with files and folders than dropdown menus and dragging icons from place to place. Even the language we use is a little different. In bash scripting you will hear folders refered to as **directories**.
 
-- On a Windows machine, the WSL (Windows Subsystem for Linux) has Ubuntu, which is just a widely used flavor of Bash, installed as its default shell scripting environment. This is automatically installed on any Windows computer running Windows 10 or later.
-- To confirm this feature is enabled, check to see if developer mode is enabled in settings and WSL is turned on in Features.
-- Search Bash in Cortana to launch WSL for the first time and click “Y” when asked to install Ubuntu for the first time. [Microsoft has their own internal documentation on WSL which features additional visual resources and FAQS](https://docs.microsoft.com/en-us/windows/wsl/).
-- Create a username and password when prompted and now you are ready to do some shell scripting! These user credentials have no bearing on your Windows user name and you will be automatically logged in for future sessions.
+To enter a command into the shell, type it in where the blinking cursor appears and press `Enter` or `return` on your keyboard.
 
-### Knowledge Check 1
+### Navigation Commands
 
-What is the name of the pre-installed program used to run shell scripts on a Mac?
+Just like when you navigate any file system, it is important to know where you are, see what files and folders are in that location, and move to other locations.
 
-[( )] Together
-[( )] MSL (Mac Subsystem for Linux)
-[(X)] Terminal
-[( )] Ubuntu
-***********************************************************************
-<div class = "answer">
+**Where am I?**
 
-WSL is the Windows utility for running shell. Ubuntu is a well known Linux operating system that is often used as the default shell scripting environment.
+The command `pwd`, which stands for **p**resent **w**orking **d**irectory, will return the path through your directory system to your current location.
```

Comment:
Consider a screenshot here to make this more concrete

Reply:
added a code block, let me know what you think.